solr/core/src/test/org/apache/solr/SolrInfoBeanTest.java

```diff
@@ -64,10 +62,11 @@ public void testCallMBeanInfo() throws Exception {
     String registry = h.getCore().getCoreMetricManager().getRegistryName();
     SolrMetricsContext solrMetricsContext = new SolrMetricsContext(metricManager, registry, "foo");
     String scope = TestUtil.randomSimpleString(random(), 2, 10);
-    for(@SuppressWarnings({"rawtypes"})Class clazz : classes ) {
+    for(Class<?> clazz : classes ) {
       if( SolrInfoBean.class.isAssignableFrom( clazz ) ) {
         try {
           SolrInfoBean info = (SolrInfoBean)clazz.getConstructor().newInstance();
+          // TODO: The following line is always true?
```

Comment:
> // TODO: The following line is always true?

Yes, it is now via the https://github.com/apache/solr/commit/e58a90f18d9823404f3f4ff59bffedf31d093a0d#diff-dd8188c05fd076ad0993f85b2d05b98f1aa8fb65d41eebbcac120f741b3c7118 change but was not at the time of the `instanceof` addition in https://github.com/apache/solr/commit/e30cc70fddcdd6fddb5eedf9f38e77fcb3f33bd1#diff-04b9e9519e437e30054389ba264496f668fc8861d8879ba5513eb01961f68605 i.e. could be simplified now then.